pkg/controller/tenant/deletion/tenanted_resources_deleter.go

```diff
@@ -151,6 +152,7 @@ func (d *tenantedResourcesDeleter) Delete(tenantName string) error {
 	if finalized(tenant) {
 		return d.deleteTenant(tenant)
 	}
+
```

Comment:
extra newline

Reply:
removed.

Reply:
Funny. I removed it. Then ran "make update" and found the extra newline was back. :-D